src/services/rfqm_service.ts

```diff
@@ -0,0 +1,95 @@
+import { MarketOperation, QuoteRequestor } from '@0x/asset-swapper';
+import { BigNumber } from '@0x/utils';
+
+import { RFQT_REQUEST_MAX_RESPONSE_MS } from '../config';
+import { NULL_ADDRESS } from '../constants';
+
+export interface FetchIndicativeQuoteParams {
+    apiKey: string;
+    amount: BigNumber; // in terms of the amount the taker is selling
+    buyToken: string;
+    sellToken: string;
+    takerAddress?: string;
+}
+
+export interface FetchIndicativeQuoteResponse {
+    buyAmount: BigNumber;
+    buyTokenAddress: string;
+    price: BigNumber;
+    sellAmount: BigNumber;
+    sellTokenAddress: string;
+}
+
+/**
+ * RfqmService is the coordination layer for HTTP based RFQM flows
+ */
+export class RfqmService {
+    constructor(private readonly _quoteRequestor: QuoteRequestor) {}
+
+    /**
+     * Fetch the best indicative quote available.
+     */
+    public async fetchIndicativeQuoteAsync(params: FetchIndicativeQuoteParams): Promise<FetchIndicativeQuoteResponse> {
+        const { amount, sellToken, buyToken, apiKey } = params;
+
+        // Map params to the terminology of Quote Requestor
+        const marketOperation = MarketOperation.Sell;
+        const assetFillAmount = amount;
+        const makerToken = buyToken;
+        const takerToken = sellToken;
+
+        // Fetch quotes
+        const opts = {
+            takerAddress: NULL_ADDRESS,
+            txOrigin: NULL_ADDRESS, // TODO - set to worker registry
+            apiKey,
+            intentOnFilling: false, // TODO - safe to hardcode?
+            isIndicative: true,
+            makerEndpointMaxResponseTimeMs: RFQT_REQUEST_MAX_RESPONSE_MS,
+            nativeExclusivelyRFQ: true,
+            altRfqAssetOfferings: {}, // TODO - set this to the alt rfq asset offerings
+            isLastLook: true,
+        };
+        const indicativeQuotes = await this._quoteRequestor.requestRfqmIndicativeQuotesAsync(
```

Comment:
Where are we reading the new RFQM configuration map? we want to have a dictionary of RFQT market makers and a dictionary of RFQM market makers right? where are the latter?

Reply:
The RFQT Market Maker map and the RFQM Market Maker map are passed in to the quoteRequestor in the constructor.

QuoteRequestor is not constructed within the `RFQMService` as I am employing dependency injection. See #542 for where the construction occurs